src/main/java/gameframework/gui/GameWindow.java

```diff
@@ -10,28 +9,26 @@
 import java.awt.event.WindowEvent;
 
 import java.awt.Component;
+import java.util.ArrayList;
 
 public class GameWindow {
 
 	protected final Frame frame;
 	protected GameCanvas gameCanvas;
-
-	protected final ObservableValue<Integer> score;
-	protected final ObservableValue<Integer> life;
+	protected final GameStatusBar statusBar = new GameStatusBar();
 
 	public GameWindow(GameCanvas gameCanvas, GameConfiguration configuration,
-			ObservableValue<Integer> score, ObservableValue<Integer> life) {
```

Comment:
please keep old constructor for backward compatibility. You can mark it as `@Deprecated`